### PR TITLE
Refactor logging logic

### PR DESCRIPTION
**Pull request by justinwhall**

### CODE REVIEW

File: packages/vip-go/services/logService.js [hunk 1]
```diff
@@ -23,7 +22,6 @@ const getService = (namespace) => {
   let service = defaultService;
   let sentryConfig = {};
   let Sentry;
```
Comment (ostowe) [inline packages/vip-go/services/logService.js:24]: I'd maybe suggest still putting this outside the function body, just in case.

**Review by ostowe — APPROVED** [review]
💯

File: packages/vip-go/services/logService.js [hunk 2]
```diff
@@ -56,7 +54,7 @@ const getService = (namespace) => {
     sentryConfig = require('@irvingjs/config/sentryConfig');
   } else {
     const getSentryConfig = require('../config/getSentryConfig');
-    sentryConfig = getSentryConfig();
+    sentryConfig = require(getSentryConfig());
```
Comment (ostowe) [inline packages/vip-go/services/logService.js:57]: Could maybe rename this function to `getSentryConfigPath` as well, for your future self.

File: packages/vip-go/services/logService.js (hunk 1 above)
Comment (justinwhall) [reply packages/vip-go/services/logService.js:24]: Oh right.

**Pull request merged**